auto_dev/protocols/performatives.py

```diff
@@ -0,0 +1,56 @@
+"""Module for parsing protocol performatives."""
+
+SCALAR_MAP = {
+    "int": "conint(ge=Int32.min(), le=Int32.max())",
+    "float": "confloat(ge=Double.min(), le=Double.max())",
+    "bool": "bool",
+    "str": "str",
+    "bytes": "bytes",
+}
```

Comment:
This is for the non-custom types that are part of a performative. 

1. `int` -> `int32`
    While float is translated into a proto `double`, they decided to save on bits when it comes to integers [ref](
https://github.com/valory-xyz/open-aea/blob/26738e20ca0937b5695e43bf2267525a475d8af4/aea/protocols/generator/common.py#L69C1-L75C2).

2. We cannot use our custom `Int32` and `Double` here
    because the scaffolded `Message` class of the framework has it's own type checking logic that does not allow for anything but `int` and `float`. They need to be in range of `int32` and `double` tho, otherwise hypothesis will generate strategies that will make tests fail (during serialization).